addons/policies/bootstrap_entity_policies.json

```diff
@@ -3701,6 +3701,78 @@
                     "entity-delete"
                 ]
             }
+        },
+        {
+            "typeName": "AuthPolicy",
+            "attributes":
+            {
+                "name": "DATA_DOMAIN_AND_PRODUCT_BUSINESS_UPDATE_METADATA",
+                "qualifiedName": "DATA_DOMAIN_AND_PRODUCT_BUSINESS_UPDATE_METADATA",
+                "policyCategory": "bootstrap",
+                "policySubCategory": "default",
+                "policyServiceName": "atlas",
+                "policyType": "allow",
+                "policyPriority": 1,
+                "policyUsers":
+                [],
+                "policyGroups":
+                [],
+                "policyRoles":
+                [
+                    "$admin",
+                    "$api-token-default-access"
+                ],
+                "policyResourceCategory": "ENTITY",
+                "policyResources":
+                [
+                    "entity:*",
+                    "entity-type:DataDomain",
+                    "entity-type:DataProduct",
+                    "entity-classification:*",
+                    "entity-business-metadata:*"
+                ],
+                "policyActions":
+                [
+                    "entity-update-business-metadata"
+                ]
+            }
+        },
+        {
+            "typeName": "AuthPolicy",
+            "attributes":
+            {
+                "name": "CRUD_DATA_PRODUCT_ENTITY",
+                "qualifiedName": "CRUD_DATA_PRODUCT_ENTITY",
```

Comment:
Also, regarding name & qualifiedName let us give some generic values for both policies
I shared context with Pratham already

Reply:
Only for bootstrap_entity_policies.json